project_euler/problem_188/sol1.py

```diff
@@ -0,0 +1,53 @@
+"""
+The hyperexponentiation of a number
+Problem 188
```

Comment:
Include the project link and any resources used, such as a wikipedia article or something which points the user to understand about this topic

Reply:
Is there any project euler problem which I can use as template? I looked at several of them and used these as a template to model my solution after

Reply:
Some algorithms don't require any external resource to solve or refer to but as this is introducing some topic, people reading this code will want to know more about it.

This is all done to make life easier for future readers of the code including yourself.

Reply:
Yes, I understand that. I just want to know where to put it. After the problem statement? In a separate section? With some heading or just as plain link? That's why I was asking about an example